app/views/warehouse_reports/cas/ce_assessments/index.haml

```diff
@@ -11,6 +11,7 @@
 - else
   .text-right.mb-4
     = link_to({format: :xlsx}.merge(filter_params),{class: 'btn btn-secondary'}) do
+      %i.icon-file-excel.icon-lg
       Download Excel
```

Comment:
@eanders Is this the style you want to standardize for all download buttons? medium sized button with icon?

I prefer "Download" over "Download Excel," because our download format is the same everywhere.

Reply:
There are two patterns for downloads:
1. https://hmis-warehouse.dev.test/performance_dashboards/overview (a drop-down with various options for download)
2. https://hmis-warehouse.dev.test/warehouse_reports/cas/ce_assessments (a single button to download in some format)

I'm not really picky about what the words say ("Download" vs. "Download PDF" or "Download Excel") when there's only one button.  It would be great to get the icons on those single download buttons, and the buttons should all have `btn` and `btn-secondary` or `btn-primary` (sometimes `btn-danger`) classes.  If those bare classes are medium, then yes, we should go with those.  
On the healthcare side, I believe those tend to be `btn-sm` which is fine for now.